sky/skylet/job_lib.py

```diff
@@ -22,6 +25,62 @@
 
 _JOB_STATUS_LOCK = '~/.sky/locks/.job_{}.lock'
 
+_DB_PATH = os.path.expanduser('~/.sky/jobs.db')
+os.makedirs(pathlib.Path(_DB_PATH).parents[0], exist_ok=True)
+CONN = sqlite3.connect(_DB_PATH)
+CURSOR = CONN.cursor()
+# CREATE TABLE
+try:
+    CURSOR.execute('SELECT * FROM pending_jobs LIMIT 0')
+except sqlite3.OperationalError:
+    # Tables do not exist, create them.
+    CURSOR.execute(''' CREATE TABLE pending_jobs(
+        job_name TEXT,
```

Comment:
Why do we use `job_name` instead of `job_id` as in L111? The `job_name` can be the same for different jobs.